src/ontology/mondo-ingest.Makefile

```diff
@@ -181,11 +185,18 @@ config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: conf
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
-# Combines all tables
-config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
-	@rm config/term_exclusions.txt; rm config/exclusion_reasons.robot.template.tsv; \
-	cat config/*_term_exclusions.txt > config/term_exclusions.txt; \
-	awk '(NR == 1) || (NR == 2) || (FNR > 2)' config/*_exclusion_reasons.robot.template.tsv > config/exclusion_reasons.robot.template.tsv
+# TODO: add back these prerequisites to this target:
+# $(REPORTDIR)/%_exclusion_reasons.ttl: mirror/%.owl $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv
+$(REPORTDIR)/%_exclusion_reasons.ttl:
```

Comment:
@matentzn I believe you said you wanted the output directory changed for all of these files from `config/` to `reports/` (except for the original exclusions files created by curators; I left those in `config/`).